src/components/ReviewCard.jsx

```diff
@@ -1,10 +1,10 @@
 export default function ReviewCard({title, owner, votes,}) {
     return(
         <article className="ReviewBox">
-            <h3 className="ReviewOwner">{owner}</h3>
             <div className="ReviewBoxDetails">
-            <h4 className="ReviewTitle">{title}</h4>
             <p className="ReviewVotes">{votes}</p>
```

Comment:
I would maybe name this and currently it just displays a number which is quite confusing as the user has no idea what this number is referring to.

Reply:
yes later on i want to add icon, just currently i've forcing myself to focus on code not on style))